src/main/java/com/webonise/rbs/service/ViewBookingsServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.webonise.rbs.service;
+
+import com.webonise.rbs.entity.Booking;
+import com.webonise.rbs.repository.BookingRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional
+public class ViewBookingsServiceImpl implements ViewBookingsService {
```

Comment:
Service should be generic. Like BookingService which will contain method to view bookings, add booking , updat  booking etc.